tilelang/language/tir/entry.py

```
@@ -40,8 +40,11 @@ def prim_func(func: Optional[Callable] = None,
     def decorator_wrapper(func):
         if not inspect.isfunction(func):
             raise TypeError(f"Expect a function, but got: {func}")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)